packages/importer/test/importer.test.cjs

```
@@ -1,12 +1,15 @@
 const assert = require("node:assert/strict")
-const { test } = require("node:test")
+const { mock, test } = require("node:test")
```

ESLint

[error] Require statement not part of import statement.

(`@typescript-eslint/no-var-requires`)

---

[error] 'require' is not defined.

(no-undef)